lib/test/Test/Pos/Block/Logic/Mode.hs

```diff
@@ -87,19 +91,14 @@ import           Pos.Ssc.Class                  (SscBlock)
 import           Pos.Ssc.Class.Helpers          (SscHelpersClass)
 import           Pos.Ssc.Extra                  (SscMemTag, SscState, mkSscState)
 import           Pos.Ssc.GodTossing             (HasGtConfiguration, SscGodTossing)
-import           Pos.Txp                        (GenericTxpLocalData, TxpGlobalSettings,
+import           Pos.Txp                        (GenericTxpLocalData, MempoolExt,
+                                                 MonadTxpLocal (..), TxpGlobalSettings,
                                                  TxpHolderTag, mkTxpLocalData)
 import           Pos.Update.Context             (UpdateContext, mkUpdateContext)
 import           Pos.Util                       (Some, newInitFuture, postfixLFields)
 import           Pos.Util.LoggerName            (HasLoggerName' (..),
                                                  getLoggerNameDefault,
                                                  modifyLoggerNameDefault)
-import           Pos.WorkMode.Class             (TxpExtra_TMP)
-#ifdef WITH_EXPLORER
```

Comment:
In this module you also forgot about `-- Remove this once there's no #ifdef-ed Pos.Txp import`.